spec/unit/procto/to_proc_spec.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Procto, '.to_proc' do
+  include_context 'procto'
+
+  subject { proc.call(text) }
+
+  let(:proc) { klass.to_proc }
+
+  let(:klass) do
+    Class.new do
+      include Procto.call
+
+      def initialize(text)
+        @text = text
+      end
+
+      def call
+        "Hello #{@text}"
+      end
+    end
+  end
+
+  it { should eql('Hello world') }
```

Comment:
Isn't this expectation already established by including the shared context and can therefore be removed?